detect_variants/max_af_filter.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "MAX_AF filter"
+baseCommand: ["/usr/bin/perl", "/opt/vep/ensembl-vep/filter_vep"]
+requirements:
+    - class: InlineJavascriptRequirement
+arguments:
+    ["--format", "vcf",
+    "-o", { valueFrom: $(runtime.outdir)/annotated.max_af_filtered.vcf }]
+inputs:
+    vcf:
+        type: File
+        inputBinding:
+            prefix: "-i"
+            position: 1
+    maximum_population_allele_frequency:
+        type: float
+        inputBinding:
+            valueFrom: |
+                ${
+                    return [
+                        "--filter",
+                        [
+                            "MAX_AF", ">", inputs.maximum_population_allele_frequency
```

Comment:
I think this should be `<`, right? This may have been caused by a typo in a slack message.

Reply:
fixed